public/docs/ts/latest/quickstart.jade

```diff
@@ -235,7 +236,7 @@ code-example(format="").
 :marked
   There are three noteworthy sections of HTML:
 
-  1. We load the JavaScript libraries we need.<br/>
+  1. We load the JavaScript libraries we need. `angular2-polyfills.js` and `Rx.js` are needed by Angular 2.<br/>
```

Comment:
@wardbell do you have a nice place to explain this? It's becoming relevant since before it was just the polyfills, but now it's those plus Rx.js.